pgrx/src/datum/time_with_timezone.rs

```diff
@@ -24,7 +24,10 @@ use std::panic::{RefUnwindSafe, UnwindSafe};
 #[derive(Debug, Copy, Clone)]
 #[repr(transparent)]
 pub struct TimeWithTimeZone(pg_sys::TimeTzADT);
-const MICROSECONDS_PER_DAY: pg_sys::TimeADT = 86_400_000;
+
+// 86_400_000_000
```

Comment:
Thanks for the fix! Could you mention that it comes from the `USECS_PER_DAY` constant?